src/Pact/Types/Runtime.hs

```diff
@@ -240,6 +242,10 @@ newtype Eval e a =
     deriving (Functor,Applicative,Monad,MonadState EvalState,
                      MonadReader (EvalEnv e),MonadThrow,MonadCatch,MonadIO)
 
+instance Monoid a => Monoid (Eval e a) where
+  mempty = pure mempty
+  mappend = liftA2 mappend
```

Comment:
Is this instance necessary?

Reply:
Yes - this was defined in anticipation of having to concatenate models. What we'd like to do is have something like `Eval e Meta <> Eval e Meta = Eval e (models ++ models')` eventually, when we start solving model constraints.

Reply:
I thought you thought that was icky

Reply:
I'll remove it for now, but we will need it in the future in order to get those sweet, sweet, `mappends`.